### ISSUE TITLE
Publish AndBible in a new app store: Accrescent

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#3342]
https://support.andbible.org/scp/tickets.php?id=2771

> Accrescent is a private and secure Android app store built with modern features in mind. It aims to provide a developer-friendly platform and pleasant user experience while enforcing modern security and privacy practices and offering robust validity guarantees for installed apps.

**Comment by tuomas2** [comment]
Currently it is invite-only, so can't proceed.

**Comment by siesta5787** [comment]
Hi @tuomas2 as a privacy conscious person i have been using Accrescent.  Like you said its invite only but I reached out to them and they said they would allow list this app so you could start using it! This would probably increase visibility of the app since there are currently not many apps in there and no other bible apps. They just need the  GitHub username for the account you would use to log in to the Accrescent developer console to add to the allow list!

**Comment by tuomas2** [comment]
My Github username is tuomas2, where do I need to tell this? Can you let them know my user account? :-)

**Comment by tuomas2** [comment]
Sent them email now.

**Comment by siesta5787** [comment]
Sorry i actually passed it along and forgot to reply to you.  Haven't heard any confirmation back yet but you might be able to create the developer account now.

**Comment by Securified** [comment]
I would love to see this happen!

**Comment by tuomas2** [comment]
i haven't heard from them

**Comment by huuhaa** [comment]
@tuomas2 seems they are now starting to be more active again:

https://blog.accrescent.app/posts/progress-update-and-roadmap/
https://blog.accrescent.app/posts/the-future-of-accrescent/

So I think now, especially cause of #3531 would be time to try going forward toward Appcrecent:

https://accrescent.app/docs/guide/getting-started/new-app.html

**Comment by meichthys** [comment]
I agree. The conversation/sentiments over on F-Droid doesn't sound good for us in the long-term (even if they revert the NSFW change). 

I would feel MUCH more comfortable supporting and listing on Accrescentapp since the developer would likely be very supportive given his profile description (Taken from the github profile):

> Christian, Android developer, and cybersecurity enthusiast. A strong believer in the Oxford comma. Creator of @accrescent.

**Comment by tuomas2** [comment]
Good news! I can proceed with submitting AndBible to Accrescent. Will be doing it later this week possibly.

**Comment by tuomas2** [comment]
Release now pending review.

**Comment by tuomas2** [comment]
Released now in Accrescent. App id there is org.andbible.andbible because they require a domain that we own.
